src/frontend/replayserver.cc

```diff
@@ -135,7 +136,16 @@ int main( void )
         }
 
         if ( best_score > 0 ) { /* give client the best match */
-            cout << HTTPResponse( best_match.response() ).str();
+            string best_response = HTTPResponse( best_match.response() ).str();
+            // Remove first line if status line is first
+            int pos = best_response.find( "\n" );
+            string first_line = best_response.substr( 0, pos );
+            if ( first_line.find( "HTTP" ) == 0 ) {
+                int start_pos = first_line.find( ' ' );
+                cout << "Status: " << first_line.substr( start_pos+1, 3 );
```

Comment:
If we're going to do it this way, best to make a separate method like HTTPMessage::serialize_for_mod_stdoutreader or something (rather than modifying HTTPMessage::str()).

But let's understand how nph works first and then see if a Status: header is the right way. I really don't like the idea of Apache parsing anything that replayserver prints (including a Status header); we want the replayserver in total command of the bytes that get sent to the client.